ADD keyboard Projectd/75/Projectd_75_ iso (#21942)
qmk/qmk_firmware: keyboards/projectd/75/iso/readme.md

Reviewer: An, imgur hosted, image of this board/PCB would be very helpful 🙂

# ProjectD 75% ISO 

> ![ProjectD 75 ISO](https://imgur.com/pbGXTUdh.png)

Reviewer 2: ```suggestion

![ProjectD 75 ISO](https://imgur.com/pbGXTUd.png)
```

> A 75% keyboard, Equipped with the WestBerry Q95 microcontroller.

Reviewer: ```suggestion
A 75% keyboard with the WestBerry Q95 microcontroller integrated into PCB.
```

* Keyboard Maintainer: [GSKY](https://github.com/gksygithub)

> * Hardware Supported: projectd_75_iso

Reviewer: ```suggestion
* Hardware Supported: ProjectD 75% ISO PCB
```

# ProjectD 75% ISO 
![ProjectD 75 ISO](https://imgur.com/pbGXTUdh.png)

> 

Reviewer: tidy
```suggestion
```

A 75% keyboard, Equipped with the WestBerry Q95 microcontroller.

* Keyboard Maintainer: [GSKY](https://github.com/gksygithub)
* Hardware Supported: projectd_75_iso
* Hardware Availability: [GSKY](https://github.com/gskygithub/projectd_75_iso)

Make example for this keyboard (after setting up your build environment):

    make projectd/75/iso:default

Flashing example for this keyboard:

    make projectd/75/iso:default:flash

To reset the board into bootloader mode, do one of the following:

* Hold the Reset switch mounted under the space key after the USB cable is connected
* Hold the Esc key while connecting the USB cable (also erases persistent settings)
* Fn+Alt+Tab will reset the board to bootloader mode

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

> 

Reviewer: tidy
```suggestion
```
